.bazelrc

```diff
@@ -5,6 +5,9 @@ common --enable_workspace  # explicitly currently still needing this
 # versions found in dependency graph.
 common --check_direct_dependencies=off
 
+# Test timeouts for various levels.
+test --test_timeout=300,1800,1800,9600
```

Comment:
This does not match the removed `--test_timeout=3600,3600,3600,3600 `

Reply:
correct. Let's say it is refined.

Reply:
I thought the goal was the avoid timeouts without having to classify the test cases now.  This seems to open the door the previous problems.

Reply:
we have essentially a bimodal set of tests. The ones that are small, and the other ones. The small ones typically run in < 1min (and mostly < 1 sec actually), so I gave them 5min.

The big flow tests are large and actually can take longer than 1h, so I gave them a good chunk.

The other test kinds are not used, so I gave them something in the middle.